### PR TITLE
add cob4-5 config

### PR DESCRIPTION
**Pull request by floweisshardt**
do not merge yet. This PR is just for discussion. scan_combined vs scan_unified needs to be solved first.

### CODE REVIEW

File: cob_navigation_global/launch/amcl_node.xml [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
+  <node pkg="topic_tools" type="relay" name="scan_right_relay" args="/base_laser_right/scan scan_combined"/>
+  <node pkg="topic_tools" type="relay" name="scan_left_relay" args="/base_laser_left/scan scan_combined"/>
+
 
   <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
 
   <!-- start amcl -->
   <node pkg="amcl" type="amcl" name="amcl">
-    <remap from="scan" to="scan_unified" />
+    <remap from="scan" to="scan_combined" />
```
Comment (floweisshardt) [inline cob_navigation_global/launch/amcl_node.xml]: why does this need to be modified? can't we use scan_unified?

File: cob_navigation_global/launch/amcl_node.xml [hunk 2]
```diff
@@ -2,14 +2,16 @@
 <launch>
 
   <!-- combine front and rear scanner -->
-  <!--node pkg="topic_tools" type="relay" name="scan_front_relay" args="/base_laser_front/scan scan_combined"/>
-    <node pkg="topic_tools" type="relay" name="scan_rear_relay" args="/base_laser_rear/scan scan_combined"/-->
+  <node pkg="topic_tools" type="relay" name="scan_front_relay" args="/base_laser_front/scan scan_combined"/>
```
Comment (floweisshardt) [inline cob_navigation_global/launch/amcl_node.xml]: I think we can get rid of the relays and use the scan_combined which is configurable per robot in cob_hardware_config. @ipa-mig: any reason to keep the relays?

Comment (mgruhler) [inline cob_navigation_global/launch/amcl_node.xml]: I think we can get rid of this.

The use-case so far has been to feed amcl with all laser scans, but they have been processed individually. I don't see a reason for not using `scan_unified`...

**Comment by floweisshardt** [comment]
removed relays and reverted to scan_unified.

@ipa-mig: ready to be merged

**Comment by floweisshardt** [comment]
@ipa-mig: please merge or comment

**Comment by mgruhler** [comment]
whoops, sorry, somehow lost track of this one...

File: cob_navigation_config/cob4-5/costmap_common_params.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+inflation_radius: 0.55
+footprint: [[0.4,0.0],[0.35,0.35],[-0.2,0.4],[-0.45,0.0],[-0.2,-0.4],[0.35,-0.35]]
+
+# BEGIN VOXEL STUFF
+observation_sources: laser_scan_front laser_scan_left laser_scan_right
+laser_scan_front: {sensor_frame: base_laser_front_link, topic: /base_laser_front/scan, data_type: LaserScan, marking: true, clearing: true}
+
+laser_scan_left: {sensor_frame: base_laser_left_link, topic: /base_laser_left/scan, data_type: LaserScan, marking: true, clearing: true}
+
+laser_scan_right: {sensor_frame: base_laser_right_link, topic: /base_laser_right/scan, data_type: LaserScan, marking: true, clearing: true}
```
Comment (mgruhler) [inline cob_navigation_config/cob4-5/costmap_common_params.yaml]: this is the old costmap params layout, please use the new one: https://github.com/ipa320/cob_navigation/blob/indigo_dev/cob_navigation_config/cob4-1/costmap_common_params.yaml

**Comment by mgruhler** [comment]
Could you maybe rebase/cherry-pick to get rid of all those merge commits?

**Comment by floweisshardt** [comment]
@ipa-mig: now using new costmap layout and rebased to ipa320

**Comment by mgruhler** [comment]
@ipa-fmw thanks.

Values seem fine, has this been tested on the robot?

FR: ok

**Comment by floweisshardt** [comment]
this is exactly the same configuration as for cob4-2. cob4-2 has been tesetd with that configuration.

**Pull request merged**